lasagne/layers/transform.py

```diff
@@ -0,0 +1,208 @@
+import theano.tensor as T
+
+from .base import MergeLayer
+
+__all__ = [
+    "TransformerLayer",
+]
+
+
+class TransformerLayer(MergeLayer):
+    """
+    Spatial transformer layer
+
+    The layer applies an affine transformation on the input. The affine
+    transformation is parameterized with six learned parameters [1]_.
+    The output is interpolated with a bilinear transformation.
+
+    Parameters
+    ----------
+    input : a :class:`Layer` instance
+        The input where the affine transformation is applied. This should
+        have convolution format, i.e. (num_batch, channels, height, width).
+    localization_network: : a :class:`Layer` instance
```

Comment:
There's a colon too much (the one in `localization_network:`). Didn't see that before, sorry. Guess it might confuse Sphinx. Feel free to amend your commit if you have the time, otherwise we can leave that for some general docstring proofreading sometime.